### PR TITLE
Brushfam info

### PR DESCRIPTION
**Pull request by coreggon11**
This PR contains updated info about openbrush, and we also mention other brushfam infrastructure tools

### CODE REVIEW

File: docs/wasm/sc-dev/use-openbrush.md [hunk 1]
```diff
@@ -33,25 +33,22 @@ Which standard tokens and useful contracts does it provide?
...
 
-- [OpenBrush Github repo](https://github.com/Supercolony-net/openbrush-contracts)
+- [OpenBrush Github repo](https://github.com/727-Ventures/openbrush-contracts)
 - [Official Docs](https://docs.openbrush.io/)
 - [OpenBrush website](https://openbrush.io/)
 - [Substrate Seminar (Youtube)](https://www.youtube.com/watch?v=I5OFGNVvzOc)
 - [How to use modifiers](https://medium.com/supercolony/how-to-use-modifiers-for-ink-smart-contracts-using-openbrush-7a9e53ba1c76)
 
-### `parity-common`
+You can join the [Brushfam Element channel](https://matrix.to/#/!utTuYglskDvqRRMQta:matrix.org?via=matrix.org&via=t2bot.io&via=web3.foundation) if you have any question about OpenBrush or ink! smart contracts in general.
```
Comment (hoonsubin) [inline docs/wasm/sc-dev/use-openbrush.md:50]: We generally don't allow adding SNS links directly to our docs. Could you add the link to the READ of the tools instead?

File: docs/wasm/sc-dev/tools.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## `parity-common`
+
+[`parity-common`](https://github.com/paritytech/parity-common) is a collection of crates that you can use in your ink! contracts.
+
+It offers all Ethereum types and is really useful if you want to port solidity code to ink!.
+
+## Questions?
+
+You can join the [Brushfam Element channel](https://matrix.to/#/!utTuYglskDvqRRMQta:matrix.org?via=matrix.org&via=t2bot.io&via=web3.foundation) if you have any question about some of the mentioned tools, or ink! smart contracts in general, and meet other ink! smart contracts developers.
```
Comment (hoonsubin) [inline docs/wasm/sc-dev/tools.md:31]: same case here

**Review by hoonsubin — CHANGES_REQUESTED** [review]

**Comment by coreggon11** [comment]
@hoonsubin thanks for the response, I removed the sns links

**Review by hoonsubin — APPROVED** [review]
@fiexer please review this too

**Comment by coreggon11** [comment]
@hoonsubin @fiexer Hello guys, any updates here?

**Comment by fiexer** [comment]
Blocker is on my end, I'll have it done EOW 🙏

**Comment by coreggon11** [comment]
@fiexer thanks for the reply 😄 sure, take your time

**Review by fiexer — APPROVED** [review]

**Pull request merged**